ArcGISRuntimeSDKQt_CppSamples/Geometry/CreateAndEditGeometries/CreateAndEditGeometries.cpp

```diff
@@ -226,41 +277,41 @@ void CreateAndEditGeometries::createConnections()
 {
   // Allow user to edit existing graphics by clicking on them
   connect(m_mapView, &MapQuickView::mouseClicked, this, [this](const QMouseEvent& mouseEvent)
-          {
-            if (!m_geometryEditor->isStarted())
-            {
-              m_mapView->identifyGraphicsOverlay(m_graphicsOverlay, mouseEvent.position().x(), mouseEvent.position().y(), 5 ,false);
-            }
-            emit canUndoOrRedoChanged();
-            emit elementIsSelectedChanged();
-          });
+  {
+    if (!m_geometryEditor->isStarted())
+    {
+      m_mapView->identifyGraphicsOverlay(m_graphicsOverlay, mouseEvent.position().x(), mouseEvent.position().y(), 5 ,false);
+    }
+    emit canUndoOrRedoChanged();
+    emit elementIsSelectedChanged();
+  });
```

Comment:
There were a bunch of formatting fixes here because previously the code style editor was having trouble with the clang style editor. Or something to that effect.